### ISSUE TITLE
Error 413 when uploading files larger than ~6 MB in Docker

### ISSUE THREAD

**Issue by Freddo3000** [FChannel0/Server#71]
Running behind a [nginx reverse proxy](https://github.com/nginx-proxy/nginx-proxy) I'm getting a `413 request entity too large` error code when trying to upload files above ~6 MB. I've tried setting `client_max_body_size` to either disabled, or way above the size of the file I'm trying to upload, which leads me to believe there might be a problem either with FChannel or the docker image.

For reference changing `client_max_body_size` has solved the issue on other services, it is only FChannel that is encountering issues.

[edit]
Connecting to the server directly yields the same result.

**Comment by ghost** [comment]
Dumb question, just checking, did you restart nginx after you edit the nginx config file?

**Comment by Freddo3000** [comment]
> Dumb question, just checking, did you restart nginx after you edit the nginx config file?

Yes, and tried manually reloading it. Anyhow working on #72 I've confirmed that there's an issue with the FChannel code somewhere.
